DirectEmployers/MyJobs
# Add command for creating Outreach Records

## What's this PR do?

Adds a management command `make_outreach_records`. It dumps some random data into outreach records so you can test NUO.

## Where should the reviewer start?

There's only one file. Note that it's all random gibberish. This should be fine since there's not really much user interaction with the data itself. It's only displayed.

## How should this be manually tested?

Clear your outreach records table.

```
$ dkm shell
>>> # completely untested
>>> from mypartners.models import OutreachRecord

>>> OutreachRecord.objects.delete()
>>> # Oh yeah. Don't forget to make sure this isn't production.
>>> [ctrl-d]
$ dkm make_outreach_records
```

Go see if you have new random data in NUO.
